### PR TITLE
Fixing kubectl cheatsheet run command comment to be more accurate

### PR DESCRIPTION
**Pull request by prabhsimransingh**
For Issue: https://github.com/kubernetes/website/issues/29278 

Fixed the comment on the kubectl cheat sheet to be more accurate. Let me know if any concerns

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ kubectl logs -f my-pod -c my-container              # stream pod container logs
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
 kubectl run nginx --image=nginx -n
```
Comment (sftim) [inline content/en/docs/reference/kubectl/cheatsheet.md:313]: aside: _this_ line should end with `\` to act as a continuation character, and I'd indent the next line to make that clear.

**Comment by prabhsimransingh** [comment]
@sftim @jimangel @krousey @zhangtreefish can you review please.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -310,8 +310,7 @@ kubectl logs -f my-pod                              # stream pod logs (stdout)
 kubectl logs -f my-pod -c my-container              # stream pod container logs (stdout, multi-container case)
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
-kubectl run nginx --image=nginx -n 
-mynamespace                                         # Run pod nginx in a specific namespace
+kubectl run nginx --image=nginx -n mynamespace      # Start a single instance of nginx pod in the namespace of mynamespace
```
Comment (reylejano) [inline content/en/docs/reference/kubectl/cheatsheet.md:313]: Just a nit.
Although correct, how about
```suggestion
kubectl run nginx --image=nginx -n mynamespace      # Run pod nginx in the "mynamespace" namespace
```
for a bit of parallelism in the examples

**Comment by reylejano** [comment]
Hi @prabhsimransingh , please see my [comment](https://github.com/kubernetes/website/pull/29879#discussion_r734899217) from 21 days ago

**Comment by jimangel** [comment]
I think the `kubectl` cheatsheet needs some cleanup in general, but that doesn't need to be addressed in this PR. For now, I think it's more accurate. Thanks for your first PR to k/website @prabhsimransingh!

/lgtm
/approve
/unhold
/label tide/merge-method-squash

**Pull request merged**